coalahtml/_coalahtml/app/views/files.html

```diff
@@ -1,9 +1,12 @@
 <div class="files">
   <div class="dir-contents" ng-if="fileType === 'dir'">
-    <h3>Contents of {{ fileName !== '' ? fileName : "[root]" }}</h3>
+    <h3>
+    Contents of <span class="filename">{{ fileName !== '' ?
+    fileName : "[root]" }}</span>
+    </h3>
     <table class="table table-striped table-bordered table-condensed"
            table-row-links>
-      <tr class="file-previous" ng-if="fileName !== ''">
+      <tr class="file-previous" ng-if="fileName.split('/').slice(-1)[0] !== ''">
```

Comment:
Maybe `.split('/').slice(-1)[0]` should be in a helper function, you use 3 times the exact same code and meaningful naming this operation would be nice.